apps/climatemappedafrica/src/components/HURUmap/KeyMetric/index.js

```diff
@@ -67,7 +66,12 @@ function KeyMetric({
           {parentValue}
         </Typography>
       )}
-      <Source href={url} classes={{ root: classes.source }}>
+      <Source
+        href={url}
+        sx={(theme) => ({
+          margin: `${theme.typography.pxToRem(20)} 0`,
```

Comment:
The useStyles you're replacing says `margin-top: 10`
```suggestion
          marginTop: typography.pxToRem(10),
```